pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -211,11 +214,17 @@ func (c *Cluster) makeMonPod(monConfig *monConfig) *v1.Pod {
 		},
 		Spec: podSpec,
 	}
+	addExtraLabels(pod.ObjectMeta.Labels)
 	cephv1.GetMonAnnotations(c.spec.Annotations).ApplyToObjectMeta(&pod.ObjectMeta)
 
 	return pod
 }
 
+func addExtraLabels(labels map[string]string) {
```

Comment:
Why do we need `addExtraLabels()`? Everything in the `getLabels()` method will be added to the deployment and pod specs already for both the mons and canaries.